### PR TITLE
Add codemirror search plugin

### LINKED ISSUE #1696: Search (ctrl+f) works only in view field in CodeMirror
No such problem in textarea. But CodeMirror is default.
Possible to fix?

### PR DESCRIPTION
**Pull request by Hirse**
# Description
- [x] Source branch in your fork has meaningful name (not `master`)
Adds the required addons for the codemirror search on the website.

Fixes Issue: 
closes #1696

# Before Merge Checklist 
These items can be completed after PR is created.

(Check any items that are not applicable (NA) for this PR)

- [x] JavaScript implementation
- [x] Python implementation (NA if HTML beautifier)
- [x] Added Tests to data file(s)
- [x] Added command-line option(s) (NA if
- [x] README.md documents new feature/option(s)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,6 +47,11 @@
   <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.48.2/mode/css/css.min.js" integrity="sha256-mjhvNBMExwa2AtP0mBlK9NkzJ7sgRSyZdgw9sPhhtb0=" crossorigin="anonymous"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.48.2/mode/htmlmixed/htmlmixed.min.js" integrity="sha256-qfS6ZUe6JhPU75/Sc1ftiWzC2N9IxGEjlRwpKB78Ico=" crossorigin="anonymous"></script>
 
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.48.4/addon/search/search.js" integrity="sha256-iUnNlgkrU5Jj8oKl2zBBCTmESI2xpXwZrTX+arxSEKc=" crossorigin="anonymous"></script>
```
Comment (bitwiseman) [inline index.html]: Please update the reset of the code mirror links to matching version.

Comment (bitwiseman) [reply index.html]: @Hirse 
Have any time to finish this up?

Comment (Hirse) [reply index.html]: Yes, absolutely. I'll have time tomorrow.

Comment (Hirse) [reply index.html]: Done. Sorry for the delay.

**Pull request merged**